### PR TITLE
Add DocumentType 

### PR DESCRIPTION
**Pull request by freddyamsterdam**

### CODE REVIEW

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+namespace Picqer\Financials\Exact;
+
+/**
+ * Class DocumentType
+ *
+ * @package Picqer\Financials\Exact
+ * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=DocumentsDocumentTypes
+ *
+ * @property String $ID Primary key
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property string $ID Primary key
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * @property DateTime $Created Creation date
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property string $Created Creation date
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ * @property String $Description Document type description
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property string $Description Document type description
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ * @property Boolean $DocumentIsCreatable Indicates if documents of this type can be created
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property bool $DocumentIsCreatable Indicates if documents of this type can be created
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+ * @property Boolean $DocumentIsDeletable Indicates if documents of this type can be deleted
+ * @property Boolean $DocumentIsUpdatable Indicates if documents of this type can be updated
+ * @property Boolean $DocumentIsViewable Indicates if documents of this type can be retrieved
+ * @property DateTime $Modified Last modified date
+ * @property Int32 $TypeCategory ID of the document type category
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property int $TypeCategory ID of the document type category
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+ * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=DocumentsDocumentTypes
+ *
+ * @property String $ID Primary key
+ * @property DateTime $Created Creation date
+ * @property String $Description Document type description
+ * @property Boolean $DocumentIsCreatable Indicates if documents of this type can be created
+ * @property Boolean $DocumentIsDeletable Indicates if documents of this type can be deleted
+ * @property Boolean $DocumentIsUpdatable Indicates if documents of this type can be updated
+ * @property Boolean $DocumentIsViewable Indicates if documents of this type can be retrieved
+ * @property DateTime $Modified Last modified date
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property string $Modified Last modified date
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+ * @package Picqer\Financials\Exact
+ * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=DocumentsDocumentTypes
+ *
+ * @property String $ID Primary key
+ * @property DateTime $Created Creation date
+ * @property String $Description Document type description
+ * @property Boolean $DocumentIsCreatable Indicates if documents of this type can be created
+ * @property Boolean $DocumentIsDeletable Indicates if documents of this type can be deleted
+ * @property Boolean $DocumentIsUpdatable Indicates if documents of this type can be updated
+ * @property Boolean $DocumentIsViewable Indicates if documents of this type can be retrieved
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property bool $DocumentIsViewable Indicates if documents of this type can be retrieved
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * @package Picqer\Financials\Exact
+ * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=DocumentsDocumentTypes
+ *
+ * @property String $ID Primary key
+ * @property DateTime $Created Creation date
+ * @property String $Description Document type description
+ * @property Boolean $DocumentIsCreatable Indicates if documents of this type can be created
+ * @property Boolean $DocumentIsDeletable Indicates if documents of this type can be deleted
+ * @property Boolean $DocumentIsUpdatable Indicates if documents of this type can be updated
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property bool $DocumentIsUpdatable Indicates if documents of this type can be updated
```

File: src/Picqer/Financials/Exact/DocumentType.php [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+ * Class DocumentType
+ *
+ * @package Picqer\Financials\Exact
+ * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=DocumentsDocumentTypes
+ *
+ * @property String $ID Primary key
+ * @property DateTime $Created Creation date
+ * @property String $Description Document type description
+ * @property Boolean $DocumentIsCreatable Indicates if documents of this type can be created
+ * @property Boolean $DocumentIsDeletable Indicates if documents of this type can be deleted
```
Comment (DannyvdSluijs) [inline src/Picqer/Financials/Exact/DocumentType.php]:
```suggestion
 * @property bool $DocumentIsDeletable Indicates if documents of this type can be deleted
```

**Comment by freddyamsterdam** [comment]
Thanks for suggestions @DannyvdSluijs, I have committed them.

**Comment by DannyvdSluijs** [comment]
@freddyamsterdam you are welcome, thank for your pull request. The user script did get some updates to generate these type hints better. I can only guess you are still using an old version, please see https://raw.githubusercontent.com/picqer/exact-php-client/master/userscript.js for the latest version for next time.

**Comment by freddyamsterdam** [comment]
yes.. and old version.. yes that's what it is. I definitely didn't miss the script all together. 👼

**Comment by freddyamsterdam** [comment]
@stephangroen could you merge please? :-)

**Pull request merged**